src/main.rs

```diff
@@ -585,80 +586,100 @@ impl ManParser for Type1 {
 //             options_section = options_section[options_matched.end()-3:]
 //             options_matched = re.search(options_parts_regex, options_section)
 
-//     def fallback(self, options_section):
-//         add_diagnostic('Trying fallback')
-//         options_parts_regex = re.compile("\.TP( \d+)?(.*?)\.TP", re.DOTALL)
-//         options_matched = re.search(options_parts_regex, options_section)
-//         if options_matched == None:
-//             add_diagnostic('Still not found')
-//             return False
-//         while options_matched != None:
-//             data = options_matched.group(2)
-//             data = remove_groff_formatting(data)
-//             data = data.strip()
-//             data = data.split("\n",1)
-//             if (len(data)>1 and len(data[1].strip())>0): # and len(data[1])<400):
-//                 optionName = data[0].strip()
-//                 if ( optionName.find("-") == -1):
-//                     add_diagnostic("%r doesn't contain '-'" % optionName)
-//                 else:
-//                     optionName = unquote_double_quotes(optionName)
-//                     optionName = unquote_single_quotes(optionName)
-//                     optionDescription = data[1].strip().replace("\n"," ")
-//                     built_command(optionName, optionDescription)
-//             else:
-//                 add_diagnostic('Unable to split option from description')
-//                 return False
-//
-//             options_section = options_section[options_matched.end()-3:]
-//             options_matched = re.search(options_parts_regex, options_section)
-//         return True
-
 impl Type1 {
-    fn fallback(&self, options_section: &str) -> bool {
-        unimplemented!()
+    fn fallback(
+        &self,
+        mut options_section: &str,
+        built_command_output: &mut Vec<String>,
+        existing_options: &mut HashSet<String>,
+        cmd_name: &str,
+    ) -> bool {
+        // add_diagnostic("Trying fallback");
+        let options_parts_re = Regex::new(r"\.TP( \d+)?((?s:.)*?)\.TP").unwrap();
```

Comment:
Should default to using `regex!` macro in case function ends up being called more than once per program run (either now or in a future refactor).

```suggestion
        let options_parts_re = regex!(r"\.TP( \d+)?((?s:.)*?)\.TP");
```

Reply:
Yeah, I did saw that macro but clippy complains about it so I am not sure to include it.